pkg/assembler/backends/ent/migrate/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM alpine:3.18 AS migrations
+
+WORKDIR /app
+
+# Copy the migration files
+COPY migrations ./migrations
+
+# Install Atlas CLI
+RUN apk add --no-cache curl && \
+    curl -sSf https://atlasgo.sh | sh
```

Comment:
Can we use the upstream image: https://atlasgo.io/guides/deploying/image#defining-the-dockerfile?
If not, can we use `go install` instead of `curl | sh`?

Reply:
So I originally had this but Atlas takes in a URL and I wanted to this work with postgres via the standard pg env variables (PGHOST PGPORT PGDATABASE PGUSER PGPASSWORD). That was the reason for creating the script. The base image `arigaio/atlas:latest`  does not contain `bin/sh` which causes the build to fail. I also was a bit iffy on the curl install but the `go install` is the better approach.

On further check there is an alpine version of ` arigaio/atlas:latest-alpine` that I can use. Updated!

Reply:
Cool, yes the entrypoint looks nice.